src/components/Common/LogoGrid.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Row, Col } from '../grid'
+import styled from 'styled-components'
+import Image from '../Common/Image'
+import ExternalAnchor from '../Common/ExternalAnchor'
+
+const Column = styled(Col)`
+  max-height: 108px;
+  display: flex;
+  align-items: center;
+  padding-bottom: ${props => props.theme.spacing[1]};
+`
+
+const LogoGrid = ({ logos = [] }) =>
```

Comment:
we shouldn't have a default value with the ternary running below.
the boolean value of [] returns true so null would never run. Remove the `= []`

Reply:
Good point. I've just copied this lazily from the original.

I'm not a big fan of the pattern where a component either renders itself or renders null. My personal preference would be for the parent to decide whether to show/hide the child or not.

For example, if `logos` don't exist in Contentful for one of the speciality pages, we don't just want the LogoGrid to disappear. We want the whole section (including the associated subtitle) to disappear. This wasn't happening previously, because we were allowing the LogoGrid to show/hide itself rather than handling it at the parent level.

I'll remove this ternary altogether :+1:

Reply:
i agree: In theory the parent should decide. Nice.